packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/pre_test.sh

```diff
@@ -14,3 +14,33 @@ test_not_val "$(agd q vstorage data published.wallet.$GOV3ADDR -o json | jq -r .
 
 # test that we have no vaults
 test_val "$(agd q vstorage data published.vaultFactory.manager0.vaults.vault0 -o json | jq -r .value)" "" "ensure no vaults exist"
+
+# provision pool has right balance
+test_val "$(agd query bank balances agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 -o json | jq -r '.balances | first | .amount ')" "19250000"
+
+
+# we should have more denoms in governance
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children | length')" "1" "more than one PSM denoms"
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children[] | select (. == "AUSD")')" "" "AUSD in IST"
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children[] | select (. == "ToyUSD")')" "" "ToyUSD in IST"
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children[] | select (. == "USDC_axl")')" "" "USDC_axl in IST"
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children[] | select (. == "USDC_grv")')" "" "USDC_grv in IST"
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children[] | select (. == "USDT_axl")')" "" "USDT_axl in IST"
+test_not_val "$(agd q vstorage children published.psm.IST -o jsonlines | jq -r '.children[] | select (. == "USDT_grv")')" "" "USDT_grv in IST"
+
+
+## testing state from pismoC was preserved post bulldozer upgrade
+
+# provision pool has right balance 
+test_val "$(agd query bank balances agoric1megzytg65cyrgzs6fvzxgrcqvwwl7ugpt62346 -o json | jq -r '.balances | first | .amount ')" "19250000"
+
+# test that the PSM gov params were preserved
+test_val "$(agoric follow -F :published.psm.IST.ToyUSD.governance -o jsonlines | jq -r '.current.MintLimit.value.value')" "$(cat /root/psm_governance.json | jq -r '.current.MintLimit.value.value')" "PSM MintLimit preserved"
```

Comment:
Not sure, it is zero, and I'm not sure on whether it's a default, it may be. I just opted to check against whatever is set in PSM as opposed to hardcode since it can change if we wire that up.